### DISCUSSION TITLE
커머스솔루션마켓 입점 문의

### DISCUSSION THREAD

**Discussion by sjhongcapbox** [commerce-api-naver/commerce-solutions#27] [묻고 답하기]
안녕하세요, 기존에 사용자들에게 직접 API키를 받는 방법으로 서비스를 하다가, 최근에 파트너API를 통하여 사용자들에게 서비스를 하고 있습니다.

그런데 최근에 생긴 커머스솔루션마켓에도 입점을 하고자 하는데요, 이곳 FAQ를 찾아봐도 입점방법이나 절차에 관하여 설명이 없어서 질문을 남깁니다.

관련하여 상세하게 안내된 링크나 문서가 있다면 부탁드립니다.

감사합니다.

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

공개적인 신규 커머스솔루션 연동과 심사 요청 절차는 내부적으로 준비중에 있습니다.
현재는 일부 개발사를 대상으로 이와 관련한 비공개 베타 테스트(CBT)를 진행중으로 내부적인 준비가 완료되는대로 상세한 내용을 커머스API센터 공지사항으로 알려드릴 예정입니다.

그 이전 커머스솔루션 연동과 관련한 사업상의 문의가 있으시다면 아래 네이버 제휴제안을 통해 내용을 남겨주시면 관련 담당자 분을 통해 별도로 연락드리도록 하겠습니다.
- https://www.navercorp.com/naver/proposalRegister
